### ISSUE TITLE
PlantUML Generation fails if both edges point to "Ende": `NotExactlyTwoOutgoingEdgesError`

### ISSUE THREAD

**Issue by hf-kklein** [Hochfrequenz/rebdhuhn#99]
Affects:
* E_0459
* E_0506
* E_0408
* E_0409
* E_0410
* E_0411
* E_0415
* E_0555
* E_0556
* E_0412
* E_0416
* E_0453
* E_0518
* E_0522
* E_0026
* E_0042
* E_0043
* E_0211
* E_0243
* E_0260
* E_0261
* E_0267
* E_1010

E.g. E_0459 in FV2310:

![grafik](https://github.com/Hochfrequenz/ebdtable2graph/assets/23094997/7cb5faaf-6b9d-4a02-81d2-6e0538aa41cd)
![grafik](https://github.com/Hochfrequenz/ebdtable2graph/assets/23094997/7e2f9213-ff6f-4661-bfc8-f14dfd60957c)

This leads to a
https://github.com/Hochfrequenz/ebdtable2graph/blob/c8915eafae24777c2885995af5f06fab1a5450ab/src/ebdtable2graph/plantuml.py#L89

Either this is prevents crashes downstream in PlantUML (then we should add a comment why we raise the error) or the assertion is not as precise as it needs to be (then we should fix it).

**Comment by lord-haffi** [comment]
Imo this error is absolutely reasonable. However, if you want to have multiple edges pointing from and to the same nodes, you'll have to use `MultiDiGraph` instead of `DiGraph`. This change maybe breaks the code somewhere else, I don't know.

**Comment by hf-kklein** [comment]
tldr: this is a feature on our side to prevent creation of puml code that might lead to wrong SVGs when used in plantuml. @lord-haffi will explain in the code.

**Comment by lord-haffi** [comment]
> tldr: this is a feature on our side to prevent creation of puml code that might lead to wrong SVGs when used in plantuml. @lord-haffi will explain in the code.

No, not this. This is a bug :) Or to be more precise: Currently, unsupported. As I mentioned:

> if you want to support multiple edges pointing from and to the same nodes, you'll have to use `MultiDiGraph` instead of `DiGraph`.

What you are talking about is "Graph too complex" but has nothing to do with this issue.

**Comment by OLILHR** [comment]
PUML in 2025 😂
